sycl/include/CL/sycl.hpp

```diff
@@ -19,6 +19,7 @@
 #include <CL/sycl/aspects.hpp>
 #include <CL/sycl/atomic.hpp>
 #include <CL/sycl/backend.hpp>
+#include <CL/sycl/backend_traits.hpp>
```

Comment:
Why did you choose to add this new header?

Reply:
Would you like me to move the implementation to backend.hpp or just remove this particular line?

Reply:
Yes, I meant that maybe we'd add this to backend.hpp?

Reply:
This is going to be a small class that never changes, so it doesn't make sense to keep it in its own file I think